drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-backend/src/main/java/org/drools/workbench/screens/guided/dtable/backend/server/conversion/DataBuilder.java

```diff
@@ -0,0 +1,177 @@
+
+                            case STRING:
+
+                                if (isTheRealCellValueString(sourceColumnIndex) && cell.getStringValue() != null) {
+
+                                    xlsRow.createCell(targetColumnIndex)
+                                            .setCellValue(String.format("\"%s\"", cell.getStringValue()));
+                                } else {
+                                    /*
+                                     * Values from lists of dates or enumerations are stored as Strings.
+                                     * For this reason we check the real data type and drop the surrounding "" if the real value was not String.
+                                     */
+                                    xlsRow.createCell(targetColumnIndex)
+                                            .setCellValue(cell.getStringValue());
+                                }
+
+                                break;
+                            case NUMERIC:
+                            case NUMERIC_BIGDECIMAL:
+                            case NUMERIC_BIGINTEGER:
+                            case NUMERIC_BYTE:
+                            case NUMERIC_DOUBLE:
+                            case NUMERIC_FLOAT:
+                            case NUMERIC_INTEGER:
+                            case NUMERIC_LONG:
+                            case NUMERIC_SHORT:
+                                final Number numericValue = cell.getNumericValue();
+                                if (numericValue != null) {
+                                    xlsRow.createCell(targetColumnIndex)
+                                            .setCellValue(numericValue.toString());
+                                }
+                                break;
+                            case DATE:
+                                final Date dateValue = cell.getDateValue();
+                                if (dateValue != null) {
+
+                                    final SimpleDateFormat formatter = new SimpleDateFormat(getDateFormat());
+
+                                    xlsRow.createCell(targetColumnIndex)
+                                            .setCellValue(formatter.format(dateValue));
+                                }
+                                break;
+                            case BOOLEAN:
+                                final Boolean booleanValue = cell.getBooleanValue();
+                                if (booleanValue != null) {
+                                    xlsRow.createCell(targetColumnIndex)
+                                            .setCellValue(booleanValue);
+                                }
+                                break;
+                        }
+                    }
+                }
+                sourceColumnIndex++;
+                targetColumnIndex++;
+            }
+            rowCount++;
+        }
+    }
+
+    private String getDateFormat() {
```

Comment:
I think we shouldn't introduce this method but rather reuse `ApplicationPreferences.getDroolsDateFormat()`, what do you think?